Added FAQ to address Query issue (#1288)
google/osv.dev: docs/faq.md

## Is the API rate limited?
No. Currently there is not a limit on the API. 

## Why am I getting an error message for my Linux Kernel query?

> Currently API queries for vulnerabilities from the Linux kernel are timing out. While we debug this behavior, we are throwing an error message for all Linux Kernel queries. You can always search for vulnerabilities on [our website](https://osv.dev/list?ecosystem=Linux) in the meantime. We apologize for the inconvenience.

Reviewer: ```suggestion
Currently the results from API queries for vulnerabilities from the Linux kernel are so large they cause timeouts. [While we optimize this use case](https://github.com/google/osv.dev/issues/151), we are failing fast and returning an error message for all Linux Kernel queries. You can always search for vulnerabilities on [our website](https://osv.dev/list?ecosystem=Linux) in the meantime, [download a full export](#is-the-database-available-to-download) for offline analysis. We apologize for the inconvenience.
```
Here's a suggested embelishment.

> Currently the results from API queries for vulnerabilities from the Linux kernel are so large they cause timeouts. [While we optimize this use case](https://github.com/google/osv.dev/issues/151), we are failing fast and returning an error message for all Linux Kernel queries. You can always search for vulnerabilities on [our website](https://osv.dev/list?ecosystem=Linux) in the meantime, [download a full export](#is-the-database-available-to-download) for offline analysis. We apologize for the inconvenience.

Author: ```suggestion
Currently the results from API queries for vulnerabilities from the Linux kernel are so large they cause timeouts. [While we optimize this use case](https://github.com/google/osv.dev/issues/151), we are failing fast and returning an error message for all Linux Kernel queries. You can always search for vulnerabilities on [our website](https://osv.dev/list?ecosystem=Linux) or [download a full export](#is-the-database-available-to-download) for offline analysis. We apologize for the inconvenience.
```
